server/src/main/java/com/google/coffeehouse/servlets/GetProfileServlet.java

```diff
@@ -65,44 +89,52 @@ public GetProfileServlet(StorageHandlerApi storageHandler) {
   public GetProfileServlet() {
     super();
     this.storageHandler = new StorageHandlerApi();
+    this.verifier = new GoogleIdTokenVerifier.Builder(transport, jsonFactory).build();
   }
   
   /** 
    * Returns a {@link Person} object in JSON format from information in the database.
-   * @param request the GET request that must have a valid JSON representation of the userId to be
-   *     passed in order to fetch a person from ID in the database.  If the required "userId"
-   *     field doesn't exist, the response object will send a "400 Bad Request error". If the JSON
+   * @param request the GET request that must have a valid JSON representation of the
+   *     {@code "idToken"} of the user to be passed in order to fetch a person from ID in the
+   *     database.  If the required "userId" field doesn't exist, the response object will
+   *     send a "400 Bad Request error". If the person with the userId passed in does not
+   *     exist, the response object will send a "404 Not Found error". If the JSON
    *     body is not valid, and unable to be parsed, the response object will send a
-   *     "500 Internal Server error". If the person with the userId passed in does not exist, the
-   *     response object will send a "404 Not Found error"
+   *     "500 Internal Server error". If the user does not have a valid ID token, the
+   *     response object will send a "403 Forbidden error"
    * @param response the response from this method, will contain the object in JSON format.
-   *     If the request object has a valid JSON body without the required field "userId", the
+   *     If the request object has a valid JSON body without the required field "idToken", the
    *     response will send a "400 Bad Request error". If the request object is attempting to get a
```

Comment:
`If the request object has a valid JSON body without the required field "idToken", the response will send a "400 Bad Request error".` This doesn't seem to be the case with this code, as a missing "idToken" field will generate a `403 Forbidden error` per the tests.

Reply:
Sorry about that, I just fixed it up (and changed it in the other files I worked on as well). Thank you for pointing that out, and please do let me know if the changes I made make more sense with the behavior I described.